docs/spec/overview.md

```diff
@@ -15,6 +16,15 @@ The interfaces are ([Addressable](interfaces.md#addressable),
 [Subscribable](interfaces.md#Subscribable), [Callable](interfaces.md#callable)).
 For more details, see [Interface Contracts](interfaces.md).
 
+- A **Broker** provides the eventing mesh.
+
+- A **Trigger** describes the filtering and subscription of events from a
```

Comment:
Consider swapping the order of Trigger and Broker, since users will likely interact with Trigger more than Broker. (See Subscription and Channel below for the same pattern.)